codelabs/spark-bigquery/backfill.sh

```diff
@@ -47,7 +47,7 @@ do
     # Submit a PySpark job via the Cloud Dataproc Jobs API
     gcloud dataproc jobs submit pyspark \
         --cluster ${CLUSTER_NAME} \
-        --jars gs://spark-lib/bigquery/spark-bigquery-latest.jar \
+        --jars gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar \
```

Comment:
Notice that `gs://spark-lib/bigquery/spark-bigquery-latest_2.12.jar` is relevant for Dataproc image 1.5 only, for image 1.4 and below you need to refer to `gs://spark-lib/bigquery/spark-bigquery-latest.jar`

Reply:
This was intentional, as the codelab that refers to this uses 1.5.